Sources/MapboxNavigation/CameraOptions.swift

```diff
@@ -0,0 +1,26 @@
+import MapboxMaps
+
+extension CameraOptions {
+    
+    /**
+     Returns description of all properties in `CameraOptions`.
+     */
+    public override var description: String {
```

Comment:
Agree that `debugDescription` is more appropriate here. Since this is mostly aims to simplify debugging I'd rather keep it on Navigation SDK side. But in general having informative `description` for all Bindgen generated interfaces would be good to have.